### PR TITLE
chore: update README.md

### PR DESCRIPTION
**Pull request by huijing**
## Changes proposed in this pull request

- This PR refines the elevator pitch of what Rafiki is.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## What is Rafiki?
 
-Rafiki is open source software that allows an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable [Interledger](https://rafiki.dev/concepts/interledger-protocol/overview/) functionality on its users' accounts.
+Rafiki is open source software that provides a more efficient method for an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable Interledger functionality on its users' accounts.
```
Comment (mkurapov) [inline README.md:7]:
```suggestion
Rafiki is open source software that provides an efficient solution for an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable Interledger functionality on its users' accounts.
```
nit-picky, but "a more" implies some sort of comparison between an alternative which we don't talk about

Comment (mkurapov) [reply README.md:7]: Should we also update the landing page?

Comment (huijing) [reply README.md:7]: good point. both of them.

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 ## What is Rafiki?
 
-Rafiki is open source software that allows an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable [Interledger](https://rafiki.dev/concepts/interledger-protocol/overview/) functionality on its users' accounts.
+Rafiki is open source software that provides an efficient solution for an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable Interledger functionality on its users' accounts.
```
Comment (melissahenderson) [inline README.md:7]: Nit-picky, but Account Servicing Entity should be in lowercase.

**Review by melissahenderson — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (AlexLakatos) [reply README.md:7]: I think we need a convention here, I would have argued "Account Servicing Entity" is always capitalized.

**Review by AlexLakatos — APPROVED** [review]

**Pull request merged**